windows/wsl_distro_install_uninstall/install_wsl.yml

```diff
@@ -41,6 +62,7 @@
   when:
     - win_powershell_cmd_output.rc is defined
     - win_powershell_cmd_output.rc != 0
+    - (guest_os_product_type | lower == 'client')
```

Comment:
Only check return code for client?

Reply:
Sorry for late response. 
Change the condition to only skip result check for lower version of windows server, whose RC=1 for the WSL installation task even the WSL and distribution are installed successfully.